tools/c7n_gcp/c7n_gcp/resources/loadbalancer.py

```diff
@@ -19,7 +19,8 @@
 
```

Comment:
can you revert these white space only changes, there's another pr in progress on load balancer fixing the get methods.